documentation/RELEASE_NOTES.md

```diff
@@ -28,6 +28,13 @@ In Base 7.0.8, an update to the compress record was added to allow for certain
 algorithms to use partially filled buffers in their computations. Unfortunately,
 this broke the behaviour of the records in certain cases. This has been fixed.
 
+### Reduce symbol and macro pollution from epicsAtomic.h on WIN32
+
+`epicsAtomic.h` no longer pulls in as many unneeded declarations and macros from
+`windows.h`. Prior to this change, including `epicsAtomic.h` at the wrong time
+could result in unexpected compiler errors. Due to the nature of `windows.h`,
+some unneeded declarations are still pulled in, however the number is greatly reduced.
+
```

Comment:
... Code which needs these declarations and macros should explicitly include `windows.h` before `epicsAtomic.h`.

Reply:
Done.